lib/itamae/resource/file.rb

```diff
@@ -169,6 +176,34 @@ def send_tempfile
           f.unlink if f
         end
       end
+
+      def backup
+        return if !attributes.backup || attributes.backup <= 0
+
+        savetime = Time.now.strftime('%Y%m%d%H%M%S')
+        backup_filename = "#{attributes.path}.itamae-#{savetime}"
+        backup_path = ::File.join(BACKUP_PATH, backup_filename)
+        backup_directory = ::File.dirname(backup_path)
+
+        run_specinfra(:create_file_as_directory, backup_directory)
+        run_specinfra(:copy_file, attributes.path, backup_path)
+        Itamae.logger.info "#{attributes.path} backed up to #{backup_path}"
+
+        basename = ::File.basename(attributes.path)
+        backup_files = run_command(['ls', '-1', backup_directory])
+        backup_files = backup_files.stdout.chomp.split("\n").select { |f|
+          f.match(/\A#{basename}.itamae-[0-9]+\z/)
+        }.reverse
```

Comment:
Should I use `run_command("ls -1 #{::File.join(BACKUP_PATH, attributes.path + '.itamae-*')}")` ?
(When I use `['ls', '-1', ::File.join(BACKUP_PATH, attributes.path + '.itamae-*']` , `*` is escaped to `\*` and can't execute.)